content/docs/tech-insights/builtin-data-sources/index.md

```diff
@@ -10,21 +10,219 @@ description: Built-in Data Sources.
 
 Built-in Data Sources use integration configurations from common Backstage plugins and annotations provided by them. You can see configuration instructions for integrations on their respective documentation pages in [the Plugins & Integrations page](/docs/integrations/).
 
-**Annotations used by built-in data sources**
+You will find a list of built-in Data Sources, below, with annotations used by them and facts they currently include.
 
-* **GitHub** and **Dependabot**
-  * `github.com/project-slug`
+### Datadog Data Source
 
-* **PagerDuty**
-  * `pagerduty.com/service-id`
+Imports data from Datadog which can be used to create Tech Insights checks.
 
-* **Datadog**
-  * `datadoghq.com/slo_tag`, for retrieving SLOs 
-  * `datadoghq.com/monitor_tags`, for retrieving monitors
+**Annotations used:** 
+ * `datadoghq.com/slo_tag`, for retrieving SLOs 
+ * `datadoghq.com/monitor_tags`, for retrieving monitors
+ 
+<details>
+<summary><b> Facts </b></summary>
 
-* **Snyk**
+| Name | Description          |
+| ------- | ------------------ |
+| Slo Count  | Number of SLOs configured in datadog matched based on the annotated tag.|
+| Monitor Count | Number of monitors configured in datadog with tags matching the monitor_tags annotation. |
+
+</details>
+
+### Dependabot Alerts Data Source
+
+Generates fact data from GitHub Dependabot Alerts for the entity.
```

Comment:
I think we need to explain how these are each authenticated. e.g. does dependabot go through the GitHub app or via a token? If we don't explain here, there's no good way for the user to know.

Reply:
I added Authentication part